utils/rowgroup/rowaggregation.cpp

```diff
@@ -1054,10 +1054,15 @@ void RowAggregation::initMapData(const Row& rowIn)
                         sizeof(int128_t),
                         colOutOffset);
                 }
-                else
+                else if (fRow.getColumnWidth(colIn) <= datatypes::MAXLEGACYWIDTH)
                 {
                     fRow.setIntField(rowIn.getIntField(colIn), colOut);
                 }
+                else
+                {
+                    idbassert(0);
+                    throw std::logic_error("RowAggregation::initMapData(): DECIMAL bad length.");
```

Comment:
Just curious, why do we need idbassert(0) along with throw logic_error() here?

Reply:
If we have decimals wider then 16  bytes we assert here for them.

Reply:
I noticed this merge targets a feature branch not a release branch, is that intentional?

Reply:
Yes. The feature isn't finished yet.